pages/common/tpp.md

```diff
@@ -0,0 +1,15 @@
+# tpp
+
+> Command-Line based presentation tool.
+
+- To use `tpp` you need to have a file which the `tpp` will interpret. You can find an example here:
```

Comment:
Since this is just a man page alternative, we do not mention installation-type instructions here. If you want to add supplementary text, you can add it as a new line to the heading.